src/accelerator.rkt

```diff
@@ -91,7 +91,7 @@
   (define ctx (context vars otype itypes))
   (define impl-fn
     (or impl
-        (let ([fn (eval-progs-real (list body) ctx)])
+        (let ([fn (eval-progs-real (list body) (list ctx))])
```

Comment:
Was this a bug?

Reply:
I think so. Jackson had it in his branch and it was causing problems without it.